packages/kit/src/views/Swap/pages/components/SwapQuoteResult.tsx

```
@@ -247,28 +247,76 @@ const SwapQuoteResult = ({
+          <SizableText size="$bodyMdMedium" color="$textSubdued">
+            {intl.formatMessage({
+              id: ETranslations.limit_est_fee,
+            })}
+          </SizableText>
+          <SizableText size="$bodyMdMedium">
+            {allCostFeeFormatValue}
+          </SizableText>
+        </XStack>
+      </YStack>
+    );
+  }, [quoteResult?.fee?.isFreeNetworkFee, intl]);
```

ESLint

[error] React Hook useMemo has missing dependencies: 'allCostFeeFormatValue', 'quoteResult?.networkCostBuyAmount', 'quoteResult?.oneKeyFeeExtraInfo?.oneKeyFeeAmount', 'quoteResult?.oneKeyFeeExtraInfo?.oneKeyFeeSymbol', and 'quoteResult?.toTokenInfo?.symbol'. Either include them or remove the dependency array.

(react-hooks/exhaustive-deps)